### ISSUE TITLE
noncollinear spin dos not supported

### ISSUE THREAD

**Issue by zhubonan** [aiida-vasp/parsevasp#72]
The code seems to assume there are up to two spin channel while there can be four in noncolinear spin calculations.

**Comment by espenfl** [comment]
Yes, this is the case and needs to be fixed for non-colinear calculations.
